CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
One more question, is the release workflow requiring you to already put the updated changelog into the branch?
And did you manage to already publish it somewhere, which triggered the creation of this file?

Reply:
I tried running my tooling to generate the file, but it generated an empty file because there is no tag to look at. After the first release, it will automatically build a changelog from all the PRs. Look at this as a placeholder. If you don't want it there now (because it is empty), that's fine; I'll remove it.